README.md

```diff
@@ -184,7 +192,7 @@ NOTE: This feature is not idempotent.  If the 'pf9' user had not been created ye
 ## Running Platform9 Express
 The basic syntax for starting Platform9 Express includes a target (host group, individual host, comma-delimited list of hosts, or "all" to run all groups) and an optional flag ('-a') that instructs it to perform role deployment.
 
-Here's an example of invoking Platform9 Express against a number of hosts:
+Here's an example of invoking Platform9 Express against a number of hosts without registering them automatically to the control plane:
```

Comment:
Change control plane to management plane

Reply:
corrected